content/product/product_management/rollout_process.md

```diff
@@ -8,6 +8,37 @@ Some features have a [beta or experimental label](../beta_and_experimental_featu
 
 Product features fall into different buckets depending on how much engagement with marketing is needed. Definitions for [L1, L2, and L3 launches](../../marketing/product-marketing/marketing_launch_tiers.md) can be found in the marketing handbook.
 
+## Communications
+
+How we communicate about features is important, especially as it relates to feature roll-out.
+
+### Pre-release features
+
+It's important we communicate clearly about pre-release features so that we do not cause confusion around customers thinking features are generally available, that they aren't going to change, or even that they couldn't be taken away if we find they aren't working.
```

Comment:
```suggestion
We always aim to communicate clearly about pre-release features so that we do not cause confusion around customers thinking features are generally available, that they aren't going to change, or even that they couldn't be taken away if we find they aren't working.
```